packages/api/authentication/handlers.go

```diff
@@ -67,7 +67,7 @@ func LoginHandler(client *ethclient.Client) func(w http.ResponseWriter, r *http.
 		}
 
 		// if block timestamp is more than [MAX_BLOCK_AGE] seconds old, reject
-		if math.Abs(time.Now().UTC().Unix() - int64(block.Time()) > MAX_BLOCK_AGE {
+		if time.Now().UTC().Unix()-int64(block.Time()) > MAX_BLOCK_AGE {
```

Comment:
`time.Now()` can technically be less than `block.Time` since blockchain time is set by a miner within a set of bounds. they can chose to set a time slightly in the future